File: firebase-database/app/src/main/java/siarhei/luskanau/example/firebase/database/AppApplication.java
package siarhei.luskanau.example.firebase.database;

import android.app.Application;

import timber.log.Timber;

public class AppApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        Timber.plant(new Timber.DebugTree());
    }
}

File: fileprovider-camera/appcrop/src/main/java/siarhei/luskanau/example/camera/appcrop/AppApplication.kt
package siarhei.luskanau.example.camera.appcrop

import android.app.Application
import com.miguelbcr.ui.rx_paparazzo2.RxPaparazzo
import timber.log.Timber
import timber.log.Timber.DebugTree

class AppApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Timber.plant(DebugTree())
        RxPaparazzo.register(this)
    }
}
